docs/index.mdx

```diff
@@ -1,36 +1,42 @@
+
+- **Keeping the overview of the design system is difficult** Setting up and maintaining the structure for a [design system](/glossary/design-system) that catalogs every widget is time-consuming. As it's not the core business of a Flutter team, it also distracts the developers from actual feature development.
+
+- **UI flaws:** Testing widgets across multiple devices, themes, and settings is time-consuming and prone to errors. When changing existing widgets, it is difficult to detect all resulting UI changes to ensure that all changes look and feel as intended.
 
-Explore the capabilities of Widgetbook with our example app by visiting our [demo page](https://demo.widgetbook.io).
+## The Solution: Widgetbook
 
-## Features
+- **Build your own design system:** Widgetbook's cataloging system helps maintain a cohesive look and feel across applications and platforms, promoting a unified user experience.
 
-- 💙 Tailored for Flutter: Designed specifically for the Flutter framework, Widgetbook offers an intuitive and seamless experience.
-- 🦄 Open Source: Accessible to everyone, Widgetbook invites you to contribute and benefit from its ongoing development.
-- 🍭 Feature-Packed: Test your widgets on a wide range of devices, themes, locales, text scales, and more, ensuring adaptability and responsiveness.
-- ⚙️ Customizable: Effortlessly adjust Widgetbook's settings to suit your unique requirements.
-- 🔧 Actively Maintained: Developed by a fast-growing Flutter startup in Germany, Widgetbook stays up-to-date with best practices in the Flutter ecosystem.
-- 👩🏼‍💻 Dedicated Support: Connect with other developers and receive assistance through our [Discord](https://discord.com/invite/zT4AMStAJA) community.
+- **Build your widgets in isolation:** Implement widgets without needing to fuss with data, APIs, or business logic. Use Widgetbook's [Addons](/addons) and [Knobs](/knobs) to quickly test your widgets in all relevant configurations.
 
-But that's not all! Widgetbook offers even more features:
+<Info>
+  On top of our [open-source](https://github.com/widgetbook/widgetbook) package,
+  we've built [Widgetbook Cloud](/cloud) to help teams save more time and ensure
+  better collaboration between designers and developers.
+</Info>
 
-- 📙 Efficient Widget Cataloging: Catalog existing widgets to prevent duplication and save time and resources.
-- 📦 Isolated Development: Focus on design and functionality by developing widgets without worrying about business logic.
-- 🎛 Knobs: Customize how your team interacts with widgets by defining adjustable knobs.
-- ⚡️ Hot Reload: Quickly identify and resolve bugs with hot reloading capabilities.
-- ✨ Comprehensive Documentation: Access easy-to-read, ever-improving documentation for all your development needs. Read the docs! ➡️
+<YouTube id="Du5yVtnmNzg" />
 
-# Widgetbook Cloud: A Collaborative Platform for Streamlined Reviews
+## Customers Showcase
```

Comment:
What about the other customer showcases? Also 1K5 and LeanCode showcase how they use the Widgetbook OSS